Beta_Integrations/integration-Vertica.yml

```diff
@@ -0,0 +1,178 @@
+description: Analytic database management software
+configuration:
+- display: Host (Vertica Server URL)
+  name: url
+  defaultvalue: myhost.example.com
```

Comment:
we should add default value only if it makes sense, please remove

Reply:
wanted to add that as example under display actually, my bad